proposals/0305-homogenize-textfieldname.md

```diff
@@ -65,21 +59,35 @@ All fields that currently exist only in the `HMI_API.xml` used to also exist in
 +   <element name="turnText">
 +     <description>Turn text for turnList parameter of UpdateTurnList</description>
 +   </element>
-   <element name="navigationText">
-     <description>Navigation text for turnList parameter of UpdateTurnList</description>
-   </element>
+-  <element name="navigationText">
+-    <description>Navigation text for turnList parameter of UpdateTurnList</description>
+-  </element>
 -  <element name="notificationText">
 -    <description>Text of notification to be displayed on screen.</description>
 -  </element>
  </enum>
+ ...
+ <struct name="Turn">
+   <param name="navigationText" type="Common.TextFieldStruct" mandatory="false">
+-     <description>Uses navigationText from TextFieldStruct.</description>
++     <description>Describes the turn using TextFieldName turnText</description>
```

Comment:
```suggestion
+     <description>Describes the turn using TextFieldName `turnText`</description>
```

Reply:
applied in https://github.com/smartdevicelink/sdl_evolution/pull/1109/commits/b9b79bf8d155d30640f71d707a337fb9080f68e0